### PR TITLE
Added guide for account linking.

### PR DESCRIPTION
**Pull request by demure**
Submitting PR per our IRC conversation.

### CODE REVIEW

File: docs/linking-qrencode-howto.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Guide to linking `signald` to an existing account using `qrencode`
+
+## Requirements
+* A phone with signal and pre-existing signal account
```
Comment (thefinn93) [inline docs/linking-qrencode-howto.md]: Please capitalize Signal

File: docs/linking-qrencode-howto.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Instructions
+### Step 1:
+Ensure that `signald` is running, **and** that `signald` has finished initalizing. Look for a message similar to the following, if checking the output of `signald`:
+
+```
+23:34:14.665 [main] INFO  signald - Started signald 0.7.0+git2019-03-24rf98e1fcb.44
+```
+
+Once `signald` is ready, connect via netcat (aka `nc`):
```
Comment (thefinn93) [inline docs/linking-qrencode-howto.md]: Might want to mention that we need `netcat-openbsd`

File: docs/linking-qrencode-howto.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+```json
+{"type": "link"}
+```
+
+### Step 3:
+After a little wait you should recieve a notice in the following format:
+
+```json
+{"type":"linking_uri","data":{"uri":"tsdevice:/?uuid=o_0123456789abcedfghij&pub_key=0123456789abcedfghijklmnopqrstuvwxyz01234567"}}
```
Comment (thefinn93) [inline docs/linking-qrencode-howto.md]: Can we use an actual linking output? Here's one I just made:
```json
{"type":"linking_uri","data":{"uri":"tsdevice:/?uuid=aes8EW8nC14Xz0aV-qugFw&pub_key=BQFy%2FyfItwo4LD3wqY7LV6i4nkWIqtYA6%2BpmlnnCk7As"}}
```

Comment (thefinn93) [reply docs/linking-qrencode-howto.md]: if you do use this, make sure to update the example URI in step 4

**Comment by demure** [comment]
Updated to correct caps/typos, use live example, and give debian package name note.

**Pull request merged**